src/v/cluster/tx_gateway_frontend.h

```diff
@@ -50,11 +50,13 @@ class tx_gateway_frontend final
     ss::future<init_tm_tx_reply> init_tm_tx(
       kafka::transactional_id,
       std::chrono::milliseconds transaction_timeout_ms,
-      model::timeout_clock::duration);
+      model::timeout_clock::duration,
+      model::producer_identity);
```

Comment:
Should this be called "expected_pid" to be consistent and conveys that the client is expecting something..